packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -369,6 +369,34 @@ const styledTemplate = (opts: StyledTemplateOpts, meta: Metadata): t.Node => {
   }) as t.Node;
 };
 
+const getKeyAttributeExpression = (node: t.Expression): t.Expression | null => {
+  if (!t.isJSXElement(node)) {
+    return null;
+  }
+
+  const keyAttribute = node.openingElement.attributes.find(
+    (attribute) => t.isJSXAttribute(attribute) && attribute.name.name === 'key'
+  ) as t.JSXAttribute | undefined;
+
+  if (
+    !keyAttribute ||
+    !t.isJSXExpressionContainer(keyAttribute.value) ||
+    !t.isExpression(keyAttribute.value.expression)
+  ) {
+    return null;
```

Comment:
`return;` and `return undefined;` is used more frequently in the codebase. Either is generally more preferential to `null`, see: https://basarat.gitbook.io/typescript/recap/null-undefined#final-thoughts